app/assets/stylesheets/views/mod-center.scss

```
@@ -0,0 +1,143 @@
+@import 'variables';
+@import 'mixins';
+@import 'config/import';
```

It's different path: 

```suggestion
@import '../config/import';
```

---

Same thing for `mod-actions.scss` file.